linux/README.md

```diff
@@ -53,6 +53,12 @@ Example:
     -PJDK_ARCHITECTURE=x64
 ```
 
+#### Optional Arguments
+- `-PVENDOR=vendor_name` - Specify a custom vendor name
+- `-PVENDOR_HOMEPAGE="www.homepage.com"` - specify a custom link to vendor homepage
```

Comment:
Reading the code, the homepage has to include the protocol. Furthermore, example.com is reserved for this purpose: `-PVENDOR_HOMEPAGE="https://example.com/"`